assets/js/theme/common/carousel/utils/setTabindexes.js

```diff
@@ -1,22 +1,18 @@
 const allFocusableElementsSelector = '[href], button, input, textarea, select, details, [contenteditable="true"], [tabindex]';
 
-export default ($slides, $prevArrow, $nextArrow, actualSlide, actualSlideCount) => {
-    $slides.each((index, element) => {
-        const $element = $(element);
-        const tabIndex = $element.hasClass('slick-active') ? 0 : -1;
-        if ($element.attr('href') !== undefined) {
-            $element.attr('tabindex', tabIndex);
-        }
+export default ($slides, $prevArrow, $nextArrow, activeSlideIdx, slidesQuantity, isInfinite) => {
+    $slides.each((idx, slide) => {
+        const $slide = $(slide);
+        const tabIndex = $slide.hasClass('slick-active') ? 0 : -1;
+        if ($slide.is(allFocusableElementsSelector)) $slide.attr('tabindex', tabIndex);
```

Comment:
Why do we need to add this statement?

Reply:
This statement checks if slide is focusable, we should control its tabindex for active slides to be focusable and for inactive hidden slides not to be accessible via keyboard